packages/terra-worklist-data-grid/src/subcomponents/Cell.module.scss

```diff
@@ -0,0 +1,45 @@
+// Themes
+@import '../orion-fusion-theme/Cell.module';
+@import '../clinical-lowlight-theme/Cell.module';
+
+:local {
+  .worklist-data-grid-cell {
+    border: var(--terra-worklist-data-grid-cell-border, 1px solid #dedfe0);
+    font-weight: normal;
+    height: 100%;
+    position: relative;
+  }
+
+  .worklist-data-grid-cell-selected {
+    background-color: var(--terra-worklist-data-grid-cell-selected-background-color, #cbe7fa);
+    box-shadow: var(--terra-worklist-data-grid-cell-selected-box-shadow, inset 0 0 0 1px #0092e0);
+  }
+
+  // Create gradient background for masked cells
+  .masked {
+    background: var(--terra-worklist-data-grid-mask-background, repeating-linear-gradient(-45deg, #dedfe0, #dedfe0 3px, transparent 3px, transparent 6px));
+  }
+
+  .cell-content {
+    height: 100%;
+    padding: var(--terra-worklist-data-grid-cell-content-padding, 7px);
+    width: 100%;
+  }
+
+  .row-selection-cell {
+    margin: auto;
+
+    input {
+      display: flex;
```

Comment:
Why would we have a flexbox layout on an input field?  Could this have been missed in being removed?  It looks like your other styles are for a block display.